apps/studio/components/interfaces/App/FeaturePreview/FeaturePreviewContext.tsx

```diff
@@ -138,11 +144,19 @@ export const useFeaturePreviewModal = () => {
           return isUnifiedLogsPreviewAvailable
         case 'new-storage-ui':
           return isNewStorageUIAvailable
+        case 'security-notifications':
+          return isSecurityNotificationsAvailable
         default:
           return true
       }
     },
-    [gitlessBranchingEnabled, advisorRulesEnabled, isUnifiedLogsPreviewAvailable]
+    [
+      gitlessBranchingEnabled,
+      advisorRulesEnabled,
+      isUnifiedLogsPreviewAvailable,
+      isNewStorageUIAvailable,
+      isSecurityNotificationsAvailable,
+    ]
```

Comment:
These dependencies were missing my earlier `isNewStorageUIAvailable`. So I added that and this new `isSecurityNotificationsAvailable` too.